aya/src/sys/mod.rs

```diff
@@ -6,19 +6,17 @@ mod perf_event;
 mod fake;
 
 #[cfg(not(test))]
```

Comment:
please remove this to fix the tests

Reply:
sorry, it's my fault, the tests is clean now 😂